.pre-commit-config.yaml

```diff
@@ -5,31 +5,18 @@ default_language_version:
 
 fail_fast: false
 repos:
-- repo: https://github.com/pocc/pre-commit-hooks
-  rev: v1.3.5
+- repo: https://github.com/Takishima/cmake-pre-commit-hooks
+  rev: v1.5.3
   hooks:
-  - id: clang-format
-    args: ["-i", "--style=file"]  # Use style set in .clang-format
-    files: |
-      (?x)^(
-        ^src/.+\.(c|h)|
-        ^test/.+\.(c|h)|
-        ^python/_spglib.c
-      )$
-    exclude: src/msg_database.c  # Skip too large file to wait every time
```

Comment:
@LecrisUT Thank you for your many efforts to improve spglib's CMake build!
Can you retain this line to skip src/msg_database.c because it takes too long time to wait every time?

Reply:
Thanks for letting me know about this